fonduer/parser.py

```diff
@@ -21,6 +22,34 @@
 logger = logging.getLogger(__name__)
 
 
+class PDFPreprocessor(DocPreprocessor):
```

Comment:
I'm also taking in this preprocessor from new_parser branch, even though it is currently unused in the tutorials.